services/portal/login/loginTasks.js

```diff
@@ -21,6 +34,9 @@ module.exports = {
     if (I.seeCurrentUrlEquals(loginProps.path)) {
       console.log('Already on Login Page');
     } else {
+      if (process.env.testedEnv.includes('mickey') || process.env.testedEnv.includes('va')) {
+        await this.systemUseMsg();
```

Comment:
you can just do a `await home.do.systemUseMsg();` here within the if loop , so that we dont have duplicate code for systemUseMsg